packages/@aws-cdk/assets/lib/asset.ts

```diff
@@ -76,6 +76,20 @@ export class Asset extends cdk.Construct {
    */
   public readonly isZipArchive: boolean;
 
+  /**
+   * The CloudFormation Parameter that represents the name of the S3 Bucket
+   * this asset will actually be stored in when deploying the Stack containing this asset.
+   * Can be used to override this location in CodePipeline.
+   */
+  public readonly bucketNameParam: cdk.CfnParameter;
```

Comment:
You don't need the parameters, you need the parameter *names*, right?

Should we not expose those instead/

Reply:
Changed.